packages/neuron-ui/src/states/init/chain.ts

```diff
@@ -24,11 +24,12 @@ export const transactionState: Readonly<State.DetailedTransaction> = {
 export const chainState: Readonly<State.Chain> = {
   networkID: currentNetworkID.load(),
   connectionStatus: ConnectionStatus.Connecting,
-  syncStatus: {
+  syncEstimate: {
```

Comment:
`estimate` is just 1 of 5 fields in it, so why rename it `syncEstimate`?

Reply:
Or we should call it the `SyncState`, which comes from the `SyncStateSubject` at the backend and includes the fields: https://github.com/nervosnetwork/neuron/blob/9343de0ddaf0c60a60c852b702286c91e3ffc7c8/packages%2Fneuron-wallet%2Fsrc%2Fcontrollers%2Fsync-api.ts#L20

The main reason I rename it from `SyncStatus` is that the name collides with the other variable, https://github.com/nervosnetwork/neuron/blob/9164b23210f395a2825e3bfa57c445e62fcc7adb/packages%2Fneuron-ui%2Fsrc%2Futils%2Fenums.ts#L109, which is now part of the `SyncState`

Reply:
It's nice to be `sync state` in neuron ui and neuron wallet

Reply:
Sure. Updated